package/yast2.spec

```diff
@@ -39,6 +39,8 @@ BuildRequires:  rubygem(%{rb_default_ruby_abi}:cheetah)
 BuildRequires:  update-desktop-files
 # For running RSpec tests during build
 BuildRequires:  rubygem(%{rb_default_ruby_abi}:rspec)
+# For converting to/from punycode strings
+BuildRequires:  rubygem(%{rb_default_ruby_abi}:simpleidn)
```

Comment:
Is the package available in SLE12-SP4?

Reply:
Yes it is, It is commented in the bug.

https://build.suse.de/request/show/168697

As commented previously I already tested it manually with the rpm from IBS.